utilize_eye_in_hand_calibration.py

```diff
@@ -0,0 +1,54 @@
+"""
+Utilize the result of eye-in-hand calibration to transform (picking) point
+coordinates from the camera frame to the robot base frame.
+"""
+
+import numpy as np
+import cv2
+
+np.set_printoptions(precision=2)
+
+
+def read_transform(file_name):
+    """Read transformation matrix from a YAML file.
+
+    Args:
+        file_name: Path to the YAML file.
+
+    Returns:
+        transform: Transformation matrix.
+
+    """
+
+    file_storage = cv2.FileStorage(file_name, cv2.FILE_STORAGE_READ)
+    file_node = file_storage.getNode("PoseState")
+    transform = file_node.mat()
+    return transform
+
+
+def _main():
+
+    # Define (picking) point in camera frame
+    point_camera = np.array([81.2, 18.0, 594.6, 1])
+    print("Point coordinates in camera frame:")
+    print(point_camera)
+
+    # Reading camera pose in end-effector frame (result of eye-in-hand calibration)
+    transform_end_effector_base = read_transform("eyeInHandTransform.yaml")
+
+    # Reading end-effector pose in robot base frame
+    transform_camera_end_effector = read_transform("robotTransform.yaml")
+
+    # Computing camera pose in robot base frame
+    transform_camera_base = np.matmul(
+        transform_end_effector_base, transform_camera_end_effector
+    )
+
+    # Computing (picking) point in robot base frame
+    point_base = np.matmul(transform_camera_base, point_camera)
+    print("Point coordinates in robot base frame:")
```

Comment:
Use f-string so you combine these two prints

Reply:
done